database/postgreSQL/schema.sql

```diff
@@ -0,0 +1,25 @@
+DROP DATABASE IF EXISTS playbar;
+
+CREATE DATABASE playbar;
+
+USE playbar;
+
+CREATE TABLE songs (
+  song_id INT NOT NULL AUTO_INCREMENT,
+  length INT NOT NULL,
+  time_stamp INT DEFAULT 0,
+  like_count INT DEFAULT 0,
+  song_data_url VARCHAR (150),
+  title VARCHAR (30),
+  artist VARCHAR (30),
+  album VARCHAR (30),
+  thumbnail_url VARCHAR (150),
+  PRIMARY KEY (song_id)
+);
+
+CREATE TABLE likes (
+  id INT NOT NULL AUTO_INCREMENT,
+  user_id INT,
+  song_id INT,
```

Comment:
you might want to specify this as a foreign key constraint with certain cascade behaviour e.g. if a song is deleted the likes associated with it would also be deleted